src/main/java/ca/corefacility/bioinformatics/irida/ria/web/analysis/AnalysisController.java

```diff
@@ -1152,7 +1152,9 @@ public boolean isShared() {
 						sequenceFilePair);
 				this.sample = sampleSequencingObjectJoin.getSubject();
 			} catch (Exception e) {
-				// IGNORE: Sample no longer exists, let the UI handle displaying the results.
+				logger.error(
```

Comment:
Could you change this maybe to `debug` or `trace`? It's not really an error since these situations are expected to occur (e.g., if a user deletes a sample).

Reply:
Also, could you add the exception to the log message? In theory, some other random exception could occur while running this code (such as database connection issue) which doesn't correspond to a missing sample.